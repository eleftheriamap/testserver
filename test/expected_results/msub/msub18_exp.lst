
test/exp/test/gen/msub/msub18_exp.bin:     file format binary


Disassembly of section .data:

0000000000000000 <.data>:
   0:	914242f7 	add	x23, x23, #0x90, lsl #12
   4:	916c6063 	add	x3, x3, #0xb18, lsl #12
   8:	d10fc908 	sub	x8, x8, #0x3f2
   c:	9b03a2f0 	msub	x16, x23, x3, x8
  10:	8a000000 	and	x0, x0, x0
