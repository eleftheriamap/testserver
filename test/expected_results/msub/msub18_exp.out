Registers:
X00    = 0000000000000000
X01    = 0000000000000000
X02    = 0000000000000000
X03    = 0000000000b18000
X04    = 0000000000000000
X05    = 0000000000000000
X06    = 0000000000000000
X07    = 0000000000000000
X08    = fffffffffffffc0e
X09    = 0000000000000000
X10    = 0000000000000000
X11    = 0000000000000000
X12    = 0000000000000000
X13    = 0000000000000000
X14    = 0000000000000000
X15    = 0000000000000000
X16    = fffff9c27ffffc0e
X17    = 0000000000000000
X18    = 0000000000000000
X19    = 0000000000000000
X20    = 0000000000000000
X21    = 0000000000000000
X22    = 0000000000000000
X23    = 0000000000090000
X24    = 0000000000000000
X25    = 0000000000000000
X26    = 0000000000000000
X27    = 0000000000000000
X28    = 0000000000000000
X29    = 0000000000000000
X30    = 0000000000000000
PC     = 0000000000000010
PSTATE : -Z--
Non-zero memory:
0x00000000 : 0x914242f7
0x00000004 : 0x916c6063
0x00000008 : 0xd10fc908
0x0000000c : 0x9b03a2f0
0x00000010 : 0x8a000000
